src/compiler/parser.ts

```diff
@@ -10022,7 +10021,7 @@ namespace IncrementalParser {
         }
 
         function visitArray(array: IncrementalNodeArray) {
-            array._children = undefined;
+            setNodeChildren(array, undefined);

```

Comment:
Setting children on arrays appears to be pointless; nothing would ever have been able to read them, so this is actually probably a perf improvement itself.

Reply:
#58069